docs/policy-provider.md

```diff
@@ -0,0 +1,50 @@
+# Policy Providers
+
+Ratify implements an extensible policy provider interface allowing for different policy providers to be created and registered. The policy provider to be used is determined by the policy plugin specified in the `policies` section of the configuration. 
+
+Currently, Ratify supports a Configuration based Policy Provider named `configPolicy`.
+
+## Sample Config
+
+```
+...
+"policies": {
+    "version": "1.0.0",
+    "plugin": {
```

Comment:
adding a comment to make sure we are updating the config at https://github.com/deislabs/ratify-action/blob/main/config.json